VirtualFleetDocker.py

```diff
@@ -0,0 +1,351 @@
+import os
+import sys
+import json
+import time
+import socket
+import shutil
+import logging
+import pathlib
+import argparse
+import multiprocessing
+from concurrent.futures import ThreadPoolExecutor
+
+import docker
+from docker.models.containers import Container
+
+NUMBER_OF_LOG_LAST_LINES_TO_SHOW = 5
+CONFIG_DIR = "./config"
+TMP_CONFIG_DIR = os.path.join(CONFIG_DIR, "tmp-configs")
+EXTERNAL_SERVER_CONFIG = os.path.join(CONFIG_DIR, "external-server/config.json")
+MODULE_GATEWAY_CONFIG = os.path.join(CONFIG_DIR, "module-gateway/config.json")
+VERNEMQ_CONFIG = os.path.join(CONFIG_DIR, "vernemq")
+LOGS_DIR = "logs"
+
+running_containers: list[Container] = []
+container_id_name_dictionary: dict[str, str] = {}
+
+
+class FileDoesntExistException(Exception):
+    pass
+
+
+class PortOutOfRangeException(Exception):
+    def __init__(self, message="Ports out of range, it has to be: 1024 < my port < 65535"):
+        super().__init__(message)
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config",
+        default=os.path.join(CONFIG_DIR, "virtual-fleet-config.json"),
+        help="path to json configuration file",
+    )
+    return parser.parse_args()
+
+
+def check_paths(arguments):
+    if not os.path.exists(arguments.config):
+        raise FileDoesntExistException(f"Json file {arguments.config} doesn't exist")
```

Comment:
Remove along with `process_arguments`.
In main before try except add:
```python
    args = parse_arguments()
    with open(args.config) as cfg_in:
        config = json.load(cfg_in)
```
Its ok to have unhandled exception for file opening, this is a simple script (end the exception is verbose enough). This little move has removed two unnecessary functions and unnecessary exception.